ACadSharp/Entities/Leader.cs

```diff
@@ -106,7 +106,12 @@ public DimensionStyle Style
 
 		//77	Color to use if leader's DIMCLRD = BYBLOCK
 
-		//340	Hard reference to associated annotation(mtext, tolerance, or insert entity)
+		/// <summary>
+		/// Hard reference to associated annotation (mtext, tolerance, or insert entity)
+		/// </summary>
+		[DxfCodeValue(340)]
```

Comment:
if this is a handle reference, please add the `DxfReferenceType.Handle` as a parameter of the `DxfCodeValue` attribute, thanks.